samwise.js

```diff
@@ -2,85 +2,88 @@
  * Autoplay plugin - Automatically advance slideshow after N seconds
```

Comment:
SCRIPT-This block of code could benefit from some comments explaining the logic behind it.